sbt-dotty/sbt-test/source-dependencies/macro-expansion-dependencies-3/changes/TestB.scala

```diff
@@ -0,0 +1,6 @@
+class B {
+  import Macros._
+
+  def f: Int = 3
+  def g: A = foo // comment out `g` first, then enable `g`
```

Comment:
> @liufengyun the incremental compilation example actually works (or fails as expected).

That might be related to a problem in my local testing. Instead of commenting it out initially, I guess changing its body to `???` will break incremental compilation.

Reply:
Still works as expected. The only way I have found to trick the incremental compiler is by reflective method calls in the implementation of the macro. #7362 can fix that.

Reply:
I'm surprised it still works.